<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
  <head>
    <meta charset="utf-8" />
    <title>exception &#8212; zfit 0.1.dev139+g3358391 documentation</title>
    <link rel="stylesheet" href="../_static/bootstrap-sphinx.css" type="text/css" />
    <link rel="stylesheet" href="../_static/pygments.css" type="text/css" />
    <script type="text/javascript" id="documentation_options" data-url_root="../" src="../_static/documentation_options.js"></script>
    <script type="text/javascript" src="../_static/jquery.js"></script>
    <script type="text/javascript" src="../_static/underscore.js"></script>
    <script type="text/javascript" src="../_static/doctools.js"></script>
    <script type="text/javascript" src="../_static/language_data.js"></script>
    <script async="async" type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="execution" href="zfit.util.execution.html" />
    <link rel="prev" title="diverse" href="zfit.util.diverse.html" />
<meta charset='utf-8'>
<meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'>
<meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1'>
<meta name="apple-mobile-web-app-capable" content="yes">
<script type="text/javascript" src="../_static/js/jquery-1.11.0.min.js "></script>
<script type="text/javascript" src="../_static/js/jquery-fix.js "></script>
<script type="text/javascript" src="../_static/bootstrap-2.3.2/js/bootstrap.min.js "></script>
<script type="text/javascript" src="../_static/bootstrap-sphinx.js "></script>

  </head><body>

  <div id="navbar" class="navbar navbar-fixed-top">
    <div class="navbar-inner">
      <div class="container">
        <!-- .btn-navbar is used as the toggle for collapsed navbar content -->
        <button class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </button>

        <a class="brand" href="../index.html">
          zfit</a>
        <span class="navbar-text pull-left"><b>0.1.dev139+g3358391</b></span>

        <div class="nav-collapse">
          <ul class="nav">
            <li class="divider-vertical"></li>
            
                <li><a href="../getting_started.html">Getting started</a></li>
                <li><a href="../space.html">Space</a></li>
                <li><a href="../parameter.html">Parameter</a></li>
                <li><a href="../model.html">Model</a></li>
                <li><a href="../data.html">Data</a></li>
                <li><a href="../loss.html">Loss</a></li>
                <li><a href="../minimize.html">Minimize</a></li>
                <li><a href="../API.html">API</a></li>
            
            
              <li class="dropdown globaltoc-container">
  <a role="button"
     id="dLabelGlobalToc"
     data-toggle="dropdown"
     data-target="#"
     href="../index.html">Site <b class="caret"></b></a>
  <ul class="dropdown-menu globaltoc"
      role="menu"
      aria-labelledby="dLabelGlobalToc"><ul>
<li class="toctree-l1"><a class="reference internal" href="../getting_started.html">Getting started with zfit</a></li>
<li class="toctree-l1"><a class="reference internal" href="../downloading.html">Downloading and Installation</a></li>
<li class="toctree-l1"><a class="reference internal" href="../contributing.html">Contributing</a></li>
<li class="toctree-l1"><a class="reference internal" href="../space.html">Space, Observable and Range</a></li>
<li class="toctree-l1"><a class="reference internal" href="../parameter.html">Parameter</a></li>
<li class="toctree-l1"><a class="reference internal" href="../model.html">Building a model</a></li>
<li class="toctree-l1"><a class="reference internal" href="../data.html">Data</a></li>
<li class="toctree-l1"><a class="reference internal" href="../loss.html">Loss</a></li>
<li class="toctree-l1"><a class="reference internal" href="../minimize.html">Minimization</a></li>
</ul>
<ul class="current">
<li class="toctree-l1 current"><a class="reference internal" href="../API.html">zfit API documentation</a></li>
</ul>
</ul>
</li>
              
            
            
            
            
            
          </ul>

          
            
<form class="navbar-form navbar-right" action="../search.html" method="get">
 <div class="form-group">
  <input type="text" name="q" class="form-control" placeholder="Search" />
 </div>
  <input type="hidden" name="check_keywords" value="yes" />
  <input type="hidden" name="area" value="default" />
</form>
          
        </div>
      </div>
    </div>
  </div>

<div class="container">
  <div class="row">
    <div class="body span12 content" role="main">
      
  <div class="section" id="module-zfit.util.exception">
<span id="exception"></span><h1>exception<a class="headerlink" href="#module-zfit.util.exception" title="Permalink to this headline">¶</a></h1>
<dl class="exception">
<dt id="zfit.util.exception.AlreadyExtendedPDFError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">AlreadyExtendedPDFError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#AlreadyExtendedPDFError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.AlreadyExtendedPDFError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.ExtendedPDFError" title="zfit.util.exception.ExtendedPDFError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.ExtendedPDFError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.AlreadyExtendedPDFError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.AlreadyExtendedPDFError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.AlreadyExtendedPDFError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.AlreadyExtendedPDFError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.AxesNotSpecifiedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">AxesNotSpecifiedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#AxesNotSpecifiedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.AxesNotSpecifiedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.NotSpecifiedError" title="zfit.util.exception.NotSpecifiedError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.NotSpecifiedError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.AxesNotSpecifiedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.AxesNotSpecifiedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.AxesNotSpecifiedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.AxesNotSpecifiedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.AxesNotUnambiguousError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">AxesNotUnambiguousError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#AxesNotUnambiguousError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.AxesNotUnambiguousError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.IntentionNotUnambiguousError" title="zfit.util.exception.IntentionNotUnambiguousError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.IntentionNotUnambiguousError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.AxesNotUnambiguousError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.AxesNotUnambiguousError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.AxesNotUnambiguousError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.AxesNotUnambiguousError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.BasePDFSubclassingError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">BasePDFSubclassingError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#BasePDFSubclassingError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.BasePDFSubclassingError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.SubclassingError" title="zfit.util.exception.SubclassingError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.SubclassingError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.BasePDFSubclassingError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.BasePDFSubclassingError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.BasePDFSubclassingError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.BasePDFSubclassingError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.ConversionError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">ConversionError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#ConversionError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.ConversionError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.ConversionError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.ConversionError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.ConversionError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.ConversionError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.DueToLazynessNotImplementedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">DueToLazynessNotImplementedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#DueToLazynessNotImplementedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.DueToLazynessNotImplementedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<p>Only for developing purpose! Does not serve as a ‘real’ Exception.</p>
<dl class="attribute">
<dt id="zfit.util.exception.DueToLazynessNotImplementedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.DueToLazynessNotImplementedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.DueToLazynessNotImplementedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.DueToLazynessNotImplementedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.ExtendedPDFError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">ExtendedPDFError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#ExtendedPDFError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.ExtendedPDFError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.ExtendedPDFError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.ExtendedPDFError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.ExtendedPDFError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.ExtendedPDFError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.IncompatibleError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">IncompatibleError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#IncompatibleError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.IncompatibleError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.IncompatibleError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.IncompatibleError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.IncompatibleError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.IncompatibleError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.IntentionNotUnambiguousError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">IntentionNotUnambiguousError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#IntentionNotUnambiguousError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.IntentionNotUnambiguousError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.IntentionNotUnambiguousError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.IntentionNotUnambiguousError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.IntentionNotUnambiguousError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.IntentionNotUnambiguousError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.LimitsIncompatibleError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">LimitsIncompatibleError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#LimitsIncompatibleError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.LimitsIncompatibleError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.IncompatibleError" title="zfit.util.exception.IncompatibleError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.IncompatibleError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.LimitsIncompatibleError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.LimitsIncompatibleError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.LimitsIncompatibleError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.LimitsIncompatibleError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.LimitsNotSpecifiedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">LimitsNotSpecifiedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#LimitsNotSpecifiedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.LimitsNotSpecifiedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.NotSpecifiedError" title="zfit.util.exception.NotSpecifiedError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.NotSpecifiedError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.LimitsNotSpecifiedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.LimitsNotSpecifiedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.LimitsNotSpecifiedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.LimitsNotSpecifiedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.LimitsOverdefinedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">LimitsOverdefinedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#LimitsOverdefinedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.LimitsOverdefinedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.OverdefinedError" title="zfit.util.exception.OverdefinedError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.OverdefinedError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.LimitsOverdefinedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.LimitsOverdefinedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.LimitsOverdefinedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.LimitsOverdefinedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.LimitsUnderdefinedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">LimitsUnderdefinedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#LimitsUnderdefinedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.LimitsUnderdefinedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.UnderdefinedError" title="zfit.util.exception.UnderdefinedError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.UnderdefinedError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.LimitsUnderdefinedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.LimitsUnderdefinedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.LimitsUnderdefinedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.LimitsUnderdefinedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.LogicalUndefinedOperationError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">LogicalUndefinedOperationError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#LogicalUndefinedOperationError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.LogicalUndefinedOperationError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.LogicalUndefinedOperationError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.LogicalUndefinedOperationError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.LogicalUndefinedOperationError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.LogicalUndefinedOperationError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.ModelIncompatibleError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">ModelIncompatibleError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#ModelIncompatibleError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.ModelIncompatibleError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.IncompatibleError" title="zfit.util.exception.IncompatibleError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.IncompatibleError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.ModelIncompatibleError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.ModelIncompatibleError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.ModelIncompatibleError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.ModelIncompatibleError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.MultipleLimitsNotImplementedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">MultipleLimitsNotImplementedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#MultipleLimitsNotImplementedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.MultipleLimitsNotImplementedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<p>Indicates that a function does not support several limits in a <a class="reference internal" href="zfit.html#zfit.Space" title="zfit.Space"><code class="xref py py-class docutils literal notranslate"><span class="pre">Space</span></code></a>.</p>
<dl class="attribute">
<dt id="zfit.util.exception.MultipleLimitsNotImplementedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.MultipleLimitsNotImplementedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.MultipleLimitsNotImplementedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.MultipleLimitsNotImplementedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.NameAlreadyTakenError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">NameAlreadyTakenError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#NameAlreadyTakenError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.NameAlreadyTakenError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.NameAlreadyTakenError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.NameAlreadyTakenError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.NameAlreadyTakenError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.NameAlreadyTakenError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.NoSessionSpecifiedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">NoSessionSpecifiedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#NoSessionSpecifiedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.NoSessionSpecifiedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.NoSessionSpecifiedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.NoSessionSpecifiedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.NoSessionSpecifiedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.NoSessionSpecifiedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.NormRangeNotImplementedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">NormRangeNotImplementedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#NormRangeNotImplementedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.NormRangeNotImplementedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<p>Indicates that a function does not support the normalization range argument <cite>norm_range</cite>.</p>
<dl class="attribute">
<dt id="zfit.util.exception.NormRangeNotImplementedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.NormRangeNotImplementedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.NormRangeNotImplementedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.NormRangeNotImplementedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.NormRangeNotSpecifiedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">NormRangeNotSpecifiedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#NormRangeNotSpecifiedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.NormRangeNotSpecifiedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.NotSpecifiedError" title="zfit.util.exception.NotSpecifiedError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.NotSpecifiedError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.NormRangeNotSpecifiedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.NormRangeNotSpecifiedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.NormRangeNotSpecifiedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.NormRangeNotSpecifiedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.NotExtendedPDFError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">NotExtendedPDFError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#NotExtendedPDFError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.NotExtendedPDFError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.ExtendedPDFError" title="zfit.util.exception.ExtendedPDFError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.ExtendedPDFError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.NotExtendedPDFError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.NotExtendedPDFError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.NotExtendedPDFError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.NotExtendedPDFError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.NotMinimizedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">NotMinimizedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#NotMinimizedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.NotMinimizedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.NotMinimizedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.NotMinimizedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.NotMinimizedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.NotMinimizedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.NotSpecifiedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">NotSpecifiedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#NotSpecifiedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.NotSpecifiedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.NotSpecifiedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.NotSpecifiedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.NotSpecifiedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.NotSpecifiedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.ObsIncompatibleError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">ObsIncompatibleError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#ObsIncompatibleError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.ObsIncompatibleError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.IncompatibleError" title="zfit.util.exception.IncompatibleError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.IncompatibleError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.ObsIncompatibleError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.ObsIncompatibleError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.ObsIncompatibleError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.ObsIncompatibleError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.ObsNotSpecifiedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">ObsNotSpecifiedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#ObsNotSpecifiedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.ObsNotSpecifiedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.NotSpecifiedError" title="zfit.util.exception.NotSpecifiedError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.NotSpecifiedError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.ObsNotSpecifiedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.ObsNotSpecifiedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.ObsNotSpecifiedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.ObsNotSpecifiedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.OverdefinedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">OverdefinedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#OverdefinedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.OverdefinedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.IntentionNotUnambiguousError" title="zfit.util.exception.IntentionNotUnambiguousError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.IntentionNotUnambiguousError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.OverdefinedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.OverdefinedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.OverdefinedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.OverdefinedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.PDFCompatibilityError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">PDFCompatibilityError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#PDFCompatibilityError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.PDFCompatibilityError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.PDFCompatibilityError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.PDFCompatibilityError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.PDFCompatibilityError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.PDFCompatibilityError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.ShapeIncompatibleError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">ShapeIncompatibleError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#ShapeIncompatibleError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.ShapeIncompatibleError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.IncompatibleError" title="zfit.util.exception.IncompatibleError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.IncompatibleError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.ShapeIncompatibleError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.ShapeIncompatibleError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.ShapeIncompatibleError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.ShapeIncompatibleError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.SpaceIncompatibleError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">SpaceIncompatibleError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#SpaceIncompatibleError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.SpaceIncompatibleError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.IncompatibleError" title="zfit.util.exception.IncompatibleError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.IncompatibleError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.SpaceIncompatibleError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.SpaceIncompatibleError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.SpaceIncompatibleError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.SpaceIncompatibleError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.SubclassingError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">SubclassingError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#SubclassingError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.SubclassingError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.SubclassingError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.SubclassingError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.SubclassingError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.SubclassingError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.UnderdefinedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">UnderdefinedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#UnderdefinedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.UnderdefinedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference internal" href="#zfit.util.exception.IntentionNotUnambiguousError" title="zfit.util.exception.IntentionNotUnambiguousError"><code class="xref py py-class docutils literal notranslate"><span class="pre">zfit.util.exception.IntentionNotUnambiguousError</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.UnderdefinedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.UnderdefinedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.UnderdefinedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.UnderdefinedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

<dl class="exception">
<dt id="zfit.util.exception.WeightsNotImplementedError">
<em class="property">exception </em><code class="sig-prename descclassname">zfit.util.exception.</code><code class="sig-name descname">WeightsNotImplementedError</code><a class="reference internal" href="../_modules/zfit/util/exception.html#WeightsNotImplementedError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#zfit.util.exception.WeightsNotImplementedError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3/library/exceptions.html#Exception" title="(in Python v3.8)"><code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></a></p>
<dl class="attribute">
<dt id="zfit.util.exception.WeightsNotImplementedError.args">
<code class="sig-name descname">args</code><a class="headerlink" href="#zfit.util.exception.WeightsNotImplementedError.args" title="Permalink to this definition">¶</a></dt>
<dd></dd></dl>

<dl class="method">
<dt id="zfit.util.exception.WeightsNotImplementedError.with_traceback">
<code class="sig-name descname">with_traceback</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#zfit.util.exception.WeightsNotImplementedError.with_traceback" title="Permalink to this definition">¶</a></dt>
<dd><p>Exception.with_traceback(tb) –
set self.__traceback__ to tb and return self.</p>
</dd></dl>

</dd></dl>

</div>


    </div>
      
  </div>
</div>
<footer class="footer">
  <div class="container">
    <p class="pull-right">
      <a href="#">Back to top</a>
      
    </p>
    <p>
        &copy; Copyright Copyright 2018, zfit.<br/>
      Created using <a href="http://sphinx-doc.org/">Sphinx</a> 2.2.1.<br/>
    </p>
  </div>
</footer>
  </body>
</html>